src/transformers/generation_utils.py

```diff
@@ -407,6 +474,13 @@ def generate(
                 conditioned on the previously generated tokens :obj:`inputs_ids` and the batch ID :obj:`batch_id`. This
                 argument is useful for constrained generation conditioned on the prefix, as described in
                 `Autoregressive Entity Retrieval <https://arxiv.org/abs/2010.00904>`__.
+            output_attentions (:obj:`bool`, `optional`, defaults to `False`):
+                Whether to return attention weights in output
+            output_hidden_states (:obj:`bool`, `optional`, defaults to `False`):
+                Whether to return hidden states in output
+            output_scores (:obj:`bool`, `optional`, defaults to `False`):
```

Comment:
`output_scores` is not yet in `configuration_utils.py` -> could you add below this line: https://github.com/huggingface/transformers/blob/c69d19faa87f33b886182f8bd347d3a9884cc566/src/transformers/configuration_utils.py#L57 ?